commonutils/src/test/java/io/intrepid/commonutils/util/MockedTextUtils.java

```diff
@@ -0,0 +1,28 @@
+package io.intrepid.commonutils.util;
+
+import android.text.TextUtils;
+
+import org.mockito.Mockito;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+import org.powermock.api.mockito.PowerMockito;
+
+import static org.mockito.Matchers.any;
+
+/**
+ * Mocks the {@link TextUtils} class.
+ *
+ * Created by aspaans on 5/6/16.
+ */
+public class MockedTextUtils {
+    public static void setup() {
+        PowerMockito.mockStatic(TextUtils.class);
+        Mockito.when(TextUtils.isEmpty(any(CharSequence.class))).then(new Answer<Boolean>() {
+            @Override
```

Comment:
It's more for the apps using this library than testing our lib. It's simple to mock it here, but we would need to redo the mock for every app that uses and tests around TextUtils.isEmpty() . Using powermock is annoying and usually is a sign of bad code design. For such a trivial method, we should just redefine it rather than relying on the TextUtils.